docs/cookbooks/creating_a_derived_dataset.md

```diff
@@ -235,7 +235,7 @@ And next is the workflow for our specific example:
 
 ```bash
 git checkout -b mozregression-aggregates
-git add dags.yaml dags/bqetl_internal_tooling.py sql/moz-fx-data-shared-prod/telemetry/mozregression_aggregates sql/moz-fx-data-shared-prod/org_mozilla_mozregression_derived/mozregression_aggregates_v1
+git add dags.yaml dags/bqetl_internal_tooling.py sql/moz-fx-data-shared-prod/org_mozilla_mozregression/mozregression_aggregates sql/moz-fx-data-shared-prod/org_mozilla_mozregression_derived/mozregression_aggregates_v1
```

Comment:
Is this meant to be `org_mozilla_mozregression_derived` just like the sql path?

Reply:
`org_mozilla_mozregression` is where the view is stored.